src/FluentValidation/TestHelper/ValidatorTestExtensions.cs

```diff
@@ -66,6 +79,19 @@ public static class ValidationTestExtension {
 			testValidationResult.ShouldNotHaveValidationErrorFor(expression);
 		}
 
+		public static void ShouldNotHaveValidationErrorFor<T>(this IValidator<T> validator, Expression<Func<T, T>> expression, string ruleSet)
+            where T : class, new()
+        {
+            T instance = Activator.CreateInstance<T>();
+            validator.TestValidate<T>(instance, ruleSet).ShouldHaveValidationErrorFor<T>(expression);
+        }
+
+        public static void ShouldNotHaveValidationErrorFor<T>(this IValidator<T> validator, Expression<Func<T, T>> expression, T value, string ruleSet)
+            where T : class, new()
+        {
+            validator.TestValidate<T>(value, ruleSet).ShouldNotHaveValidationErrorFor<T>(expression);

```

Comment:
Does this definitely behave correctly? I'd switch it to `ShouldNotHaveValidationErrorFor(string.Empty)` just to be completely explicit.